### PR TITLE
Refactor/python test

### PR DESCRIPTION
**Pull request by vbkaisetsu**
This branch groups tests by classes

### CODE REVIEW

File: python-primitiv/tests/device.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        dev = Device.get_default()
+        self.assertIs(self.naive_dev, dev)
+        if self.cuda_dev:
+            Device.set_default(self.cuda_dev)
+            dev = Device.get_default()
+            self.assertIs(self.cuda_dev, dev)
+
+    def test_num_devices(self):
+        if CUDA:
+            self.assertIsInstance(CUDA.num_devices(), int)
```
Comment (odashi) [inline python-primitiv/tests/device.py]: Add an assertion whether the return value is 0 or positive.

File: python-primitiv/tests/graph.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+from primitiv import Graph
+from primitiv import Shape
+from primitiv import operators as F
+from primitiv.devices import Naive
+
+import numpy as np
+import unittest
+
+
+class ArgumentTest(unittest.TestCase):
```
Comment (odashi) [inline python-primitiv/tests/graph.py]: `GraphTest`

File: python-primitiv/tests/parameter.py [hunk 3]
```diff
@@ -61,3 +81,31 @@ def test_parameter_gradient(self):
         self.assertTrue((grad.to_ndarrays()[0] == np.ones([8])).all())
         with self.assertRaises(NotImplementedError):
             del self.p.gradient
+
+    def test_parameter_argument(self):
```
Comment (odashi) [inline python-primitiv/tests/parameter.py]: Add an assertion that fails when using mini-batched data as the initializer.

**Comment by vbkaisetsu** [comment]
I want to rewrite test cases of Python, so I close this pull request.

**Pull request closed without merging**